src/config.rs

```diff
@@ -219,8 +219,14 @@ impl Config {
     }
 
     /// Returns the proxy URL if defined.
-    fn get_proxy_url(&self) -> Option<&str> {
-        self.ini.get_from(Some("http"), "proxy_url")
+    pub fn get_proxy_url(&self) -> Option<String> {
+        if env::var_os("proxy_url").is_some() {
```

Comment:
That should be `http_proxy` IIRC, can you check?
```suggestion
        if env::var_os("http_proxy").is_some() {
```

Reply:
Good catch, thanks. Also removed the env binding override, as it's not used anymore.